opensearch-operator/opensearch-gateway/responses/SnapshotRepositoryResponse.go

```diff
@@ -0,0 +1,5 @@
+package responses
+
+import "github.com/Opster/opensearch-k8s-operator/opensearch-operator/opensearch-gateway/requests"
+
+type SnapshotRepositoryResponse = map[string]requests.SnapshotRepository
```

Comment:
From you comment, I understood now this queries the OpenSearch API and saves the result in `SnapshotRepositoryResponse ` which is of `SnapshotRepository` struct format.